blog/_posts/2019-01-18-fluxdiffeq.md

```diff
@@ -0,0 +1,650 @@
+---
+layout: post
+title:  FluxDiffEq.jl: A Julia Library for Neural Differential Equations
+author: Chris Rackauckas, Mike Innes, Yingbo Ma, Lyndon White, Vaibhav Dixit
+---
+
+In this blog post we
+will show you how to easily, efficiently, and
+robustly add differential equation solvers to
+neural networks in Julia.
+
+![Flux ODE Training Animation](https://user-images.githubusercontent.com/1814174/51399500-1f4dd080-1b14-11e9-8c9d-144f93b6eac2.gif)
+
+The [Neural Ordinary Differential Equations](https://arxiv.org/abs/1806.07366)
+paper has been a hot topic even before it made a splash as Best Paper of
+NeuralIPS 2018. By mixing ordinary differential equations and neural networks
+they were able to improve training speeds and accuracy over ResNet. Now with the
+floodgates opened causing a merge of differential equations merging with machine
+learning, the purpose of this blog post is into introduce the reader to
+differential equations from a data science perspective and show how to mix these
+tools with neural nets.
+
+The advantages of the Julia [DifferentialEquations.jl](https://github.com/JuliaDiffEq/DifferentialEquations.jl) library for numerically solving
+differential equations have been
+[discussed in detail in other posts](http://www.stochasticlifestyle.com/comparison-differential-equation-solver-suites-matlab-r-julia-python-c-fortran/).
+Recently, these native Julia differential equation solvers have successfully been embedded
+into the [Flux.jl](https://github.com/FluxML/Flux.jl/) deep learning package, to allow the use of a full suite of
+highly tested and optimized DiffEq methods within neural networks. Using the new package
+[DiffEqFlux.jl](https://github.com/JuliaDiffEq/DiffEqFlux.jl/), we will show the reader how to easily add differential equation
+layers to neural networks using a range of differential equations models, including stiff ordinary
+differential equations, stochastic differential equations, delay differential
+equations, and hybrid (discontinuous) differential equations.
+
+This is the first user-friendly toolbox to combine state-of-the-art differential

```

Comment:
Can you qualify this statement with torchdiffeq? Possibly expand 'user-friendly' to distinguish benefits of DiffEq.jl.

Reply:
Maybe there's a better way of putting this together. I mean for it to kind of go with the previous line, essentially saying this is the first toolbox which combines neural networks with well-optimized and battle tested DE solvers for stiff and non-stiff ODEs, DAEs, DDEs, SDEs, etc. torchdiffeq only does the two basic explicit RK methods (and then a few low order fixed time step methods), so I would think the difference is clear. 

The libraries that would need to be qualified against more IMO are FATODE and Sundials which are also production-quality ODE solvers with adjoint sensitivity analysis implemented, but I think in their case it's clear that they aren't hooked up with neural nets (though they could be).

Reply:
See if the changed introduction satisfies you. It more qualifies the statement by listing some of the standard features of modern diffeq solver libraries, like distributed parallelism and event handling.